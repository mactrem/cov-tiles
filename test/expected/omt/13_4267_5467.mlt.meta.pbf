F
	landcover
id" 
geometry* 
class" 	
subclass" 	°
waterway
id" 
geometry* 
class" 	
intermittent" 
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
brunnel" 	2
landuse
id" 
geometry* 
class" 	¹
mountain_peak
id" 
geometry* 
class" 	
ele" 
ele:ft" 
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
rank" ”
place
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
rank" ¤
poi
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	
rank" 
subclass" 	F
water
id" 
geometry* 
class" 	
intermittent" ×
transportation_name
id" 
geometry* 
brunnel" 	
class" 	
network" 	
ref" 	

ref:length" 
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	…
park
id" 
geometry* 
class" 	
name" 	
name:de" 	
name:en" 	
name:int" 	

name:latin" 	$
building
id" 
geometry* Ù
transportation
id" 
geometry* 
brunnel" 	
class" 	
layer" 
oneway" 
ramp" 
surface" 	
service" 	
bicycle" 	
subclass" 	
foot" 	
horse" 	