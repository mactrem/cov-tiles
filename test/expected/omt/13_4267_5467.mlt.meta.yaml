version: 1
featureTables {
  name: "landcover"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "waterway"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "intermittent"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "brunnel"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "landuse"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "mountain_peak"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ele"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "ele:ft"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "place"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "poi"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_64
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "rank"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "water"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "intermittent"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
}
featureTables {
  name: "transportation_name"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "brunnel"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "network"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "ref:length"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "park"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:de"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:en"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:int"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "name:latin"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
featureTables {
  name: "building"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
}
featureTables {
  name: "transportation"
  columns {
    name: "id"
    scalarType {
      physicalType: UINT_32
    }
  }
  columns {
    name: "geometry"
    complexType {
      physicalType: GEOMETRY
    }
  }
  columns {
    name: "brunnel"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "class"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "layer"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "oneway"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "ramp"
    nullable: true
    scalarType {
      physicalType: INT_64
    }
  }
  columns {
    name: "surface"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "service"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "bicycle"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "subclass"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "foot"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
  columns {
    name: "horse"
    nullable: true
    scalarType {
      physicalType: STRING
    }
  }
}
